test/fixtures/dbt_project/README.md

```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
```

Comment:
We probably don't need this file right?

Reply:
I'll remove it :) @sethwoodworth also made a comment about slimming down the DBT project so I'll remove as much as possible